CMSgov/onemac
# OY2 22496: Default Title "Appendix K Amendment" displayed instead of the given title for the Amendment Title field in the details page for App K Waivers.

Story: https://qmacbis.atlassian.net/browse/OY2-22496
Endpoint: See github-actions bot comment

### Details
The Appendix K Amendment Title was not getting saved as an attribute on the package

### Changes
- Added title as an attribute in the Appendix K attribute list.
- did not find any other packages using Title

### Test Plan
1. Login as a submitting user
2. Navigate to Package Dashboard
3. Use New Submission->Waiver Action->1915(c) Appendix K Amendment to start an Appendix K Form
4. Submit the form with a title
5. save the Appendix K
6. Verify the custom title shows in the detail page for that package.
